Dockerfile

```diff
@@ -15,6 +17,11 @@ LABEL io.k8s.description="ASP.NET 5.0" \
 
 RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
```

Comment:
@jawnsy I debated about this but opted to follow MS' example.   I'm fine to collapse.